docs/guides/htpc.md

```diff
@@ -119,23 +167,23 @@ vo=gpu-next
 target-contrast=inf
 target-trc=pq
 target-prim=bt.2020
-target-peak=700    ##IF YOU HAVE AN HDR TV, ADJUST THIS TO THE 10% PEAK
+target-peak=700    ## If you have an HDR display, adjust this to the 10% peak
 
-## Above makes use of vo=gpu-next for HDR Content, and vo=gpu for SDR. This is a requirement for blend-subtitles=video.
+## Above makes use of vo=gpu-next for HDR Content, and vo=gpu for SDR. This is a requirement for blend-subtitles=video
 ```
 
 ==-
 
-In my input.conf I have a single line, for deband.
+#### Scripts
 
-```yaml
-BACK cycle deband
```

Comment:
You should leave some hint about tying deband to a key

Reply:
Resolved in https://github.com/Snaacky/thewiki/pull/166/commits/833bbf18904ae7deee475422b3b6489cd05a32a9.